problems/[230117] 20. Valid Parentheses.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/valid-parentheses/
+ *
+ * @param {string} s
+ * @return {boolean}
+ */
+const isValid = function (s) {
+  if (s.length === 1 || new Set(s).size === 1) return false;
+
+  const leftParentheses = [];
+
+  for (let char of s) {
+    if (char === "(" || char === "{" || char === "[") {
+      leftParentheses.push(char);
+    }
+
+    if (char === ")") {
+      if (leftParentheses.pop() !== "(") return false;
+    }
+
+    if (char === "}") {
+      if (leftParentheses.pop() !== "{") return false;
+    }
+
+    if (char === "]") {
+      if (leftParentheses.pop() !== "[") return false;
+    }
+  }
+
+  return leftParentheses.length ? false : true;
```

Comment:
```suggestion
  return !!leftParentheses.length; // 혹은 leftParentheses.length > 0
```

truthy 혹은 falsy값을 boolean으로 나타내기 위해서 삼항연산자보다는 다른 연산자를 쓰는 게 좋을 것 같아요!